code/scripting/api/objs/file.cpp

```diff
@@ -6,63 +6,72 @@ extern int cfread_lua_number(double *buf, CFILE *cfile);
 namespace scripting {
 namespace api {
 
+cfile_h::cfile_h(CFILE* cfp) : _cfp(cfp) {}
```

Comment:
Is implicit conversion intended here? If not I'd declare the constructor explicit.